src/TestFramework/Coverage/XmlReport/JUnitTestCaseSorter.php

```diff
@@ -63,31 +73,25 @@ static function (CoverageLineData $a, CoverageLineData $b) {
             }
         );
 
-        return array_map(
```

Comment:
One improvement I do see there is that we could do the mapping to `testFilePath` directly in `uniqueByTestFile()` to avoid a second looping

Reply:
Isn't it going to need a `time` field to sort by it later?

Reply:
damn right, nevermind... then